### ISSUE TITLE
Links to wrong version of BSD license

### ISSUE THREAD

**Issue by webchick** [simplepie/simplepie#170]
Hi there, folks. Sorry if this is the wrong place to file this issue.

I'm coming from the Drupal project, where we're attempting to catalog a list of GPL-compatible external libraries for use in our distribution packaging system.

http://www.gnu.org/licenses/license-list.html#GPLCompatibleLicenses has a big long list of licenses which are and are not GPL-compatible. One of the ones that's not is the classic 2-clause BSD license, which is unfortunately the license (http://www.opensource.org/licenses/bsd-license.html) that  http://simplepie.org/wiki/faq/can_i_include_simplepie_with_software_i_m_selling links to, and also the one that README.md links to.

However, in the actual source code at https://github.com/simplepie/simplepie/blob/master/LICENSE.txt shows it's /actually/ the 3-clause BSD license which is at http://www.opensource.org/licenses/BSD-3-Clause. The text "New BSD license" in README.md also seems to indicate this is a mistake, rather than intended.

TL;DR: Please link to http://www.opensource.org/licenses/BSD-3-Clause instead of http://www.opensource.org/licenses/bsd-license.html :)
